client/src/main/scala/com.gu.contentapi.client/model/PaginatedApiResponse.scala

```diff
@@ -1,31 +0,0 @@
-package com.gu.contentapi.client
-
-import com.gu.contentapi.client.model.v1._
-import com.gu.contentatom.thrift.Atom
-
-/** Typeclass witnessing how to extract the id for
-  * a paginated query
-  */
-trait PaginatedApiResponse[Response] {
-  /** the id for a next query, if any */
-  def getNextId: Response => Option[String]
-}
```

Comment:
This trait, and the implementations below, are no longer needed because `pageableResponseDecoder()` enables similar functionality with less repeated code - no need to repeat this code snippet below 3 times:

```
if (r.results.length < r.pageSize) None else r.results.lastOption.map(_.id)
```